### PR TITLE
Add WGC to token list

### PR DESCRIPTION
**Pull request by jameswgc**
**Description**

Adding token information for WGC on Ethereum, Optimism and Base.

Thanks

**Tests**

Please describe any tests you've added. If you've added no tests, or left important behavior untested, please explain why not.

**Additional context**

Add any other context about the problem you're solving.

**Metadata**

- Fixes #[Link to Issue]

### CODE REVIEW

**Comment by jameswgc** [comment]
Hello guys, I did see the instruction to separate the pull request and send to [@roberto-bayardo](https://github.com/roberto-bayardo). I'm not sure how to do this, think it's too late for me to do it now.
Would appreciate it if you could please merge to the token list.
Thanks

**Review by usuarrio-id81214293 — APPROVED** [review]

**Comment by jameswgc** [comment]
Hello guys, 
I've just replaced the logo.svg file with a different version. 
Logo was originally made in photoshop and we've had issues converting to SVG.
The new version just uploaded is slightly cleaner looking with a smaller file size.
Thanks

**Comment by hamdiallam** [comment]
Hi @jameswgc, It looks like this token taht isn't bridged by the standard bridge but a 3rdparty protocol. Can you add the "nobridge" tag to your entry as well as update the token description to specify the protocol responsible for bridging this token? 

No worries having both Base/Optimism in the same PR. Just needs approvals from both before we can merge them in.

**Pull request closed without merging**
